tests/h2o/test_h2o_model_export.py

```diff
@@ -19,13 +19,16 @@
 import mlflow.pyfunc.scoring_server as pyfunc_scoring_server
 from mlflow import pyfunc
 from mlflow.models import Model
+from mlflow.store.s3_artifact_repo import S3ArtifactRepository
 from mlflow.tracking.artifact_utils import _get_model_log_dir
 from mlflow.utils.environment import _mlflow_conda_env
 from mlflow.utils.file_utils import TempDir
 from mlflow.utils.model_utils import _get_flavor_configuration
 
 from tests.helper_functions import pyfunc_serve_and_score_model
```

Comment:
nit: can we just put all these together - not as pretty but more coherent

Reply:
Will follow up with a PR to do this throughout the various test suites; would like to minimize the diff here.